src/modules/commander/commander_params.c

```diff
@@ -799,9 +799,21 @@ PARAM_DEFINE_INT32(NAV_RCL_ACT, 2);
 
 /**
  * Flag to enable obstacle avoidance
- * Temporary Parameter to enable interface testing
  *
  * @boolean
  * @group Mission
  */
-PARAM_DEFINE_INT32(COM_OBS_AVOID, 0);
\ No newline at end of file
+PARAM_DEFINE_INT32(COM_OBS_AVOID, 0);
+
+/**
+ * Set onboard controller bootup timeout
+ *
+ * This parameter defines the bootup timeout.
```

Comment:
What about:

```
The avoidance system is expected to boot within this bootup timeout. If no avoidance system has been detected by this time, a MAVLink warning message is sent.
```

Reply:
Much better than what was here. See above for my version - it is much the same, but you don't need to mention time or  timeout quite so much.